packages/lit-dev-content/site/tutorials/content/carousel/05.md

```diff
@@ -0,0 +1,57 @@
+It works! But it's a little boring, and changing the selected item is a bit
+jarring. This is a great place to use animation to provide feedback about what's
+happening. Animating elements can be tricky, so we'll tackle it in a few steps.
+First, we need to modify the DOM and styling a bit to prepare for adding the
+animation.
+
+We want to slide the selected item into view from the left or right based on
+whether or not the next item is greater than the current one. To do this,
+we'll add a slot for the `previous` element. We'll use this to manage the
+positions of the previously and currently selected items. Then we'll add a
+"container" element surrounding the items that we'll animate left and right.
+We'll move our click handler there to make sure we don't miss any clicks.
+
+We're going to position `left` property of the "container" and the `previous`
+and `selected` slots. To setup for that, we've already imported
+the `styleMap` directive and we'll use it in the template. For now just
```

Comment:
stylemap is compiled away in JS example. Can simply just include the import in the code snippet below

Reply:
Added.